### ISSUE TITLE
multi_scan interprets skip_build differently than scan does

### ISSUE THREAD

**Issue by lilyball** [lyndsey-ferguson/fastlane-plugin-test_center#252]
### New Issue Checklist

- [X] Updated `fastlane-plugin-test_center` to the latest version
- [X] I read the [README.md](https://github.com/lyndsey-ferguson/fastlane-plugin-test_center/blob/master/README.md)
- [X] I reviewed the [example(s)](https://github.com/lyndsey-ferguson/fastlane-plugin-test_center/blob/master/README.md) for the action(s) I am using
- [X] I have removed any sensitive data such as passwords, authentication tokens, or anything else I do not want to world to see

> If you love this fastlane plugin, consider sponsoring it or asking your company to sponsor it. I would really appreciate any
> gesture: https://github.com/sponsors/lyndsey-ferguson. 😍

### Issue Description
By default `scan` will run `xcodebuild build test`. This means it builds twice (assuming the default configuration of CLI build as Release and test as Debug). This can be modified with `skip_build`, `test_without_building`, or `build_for_testing`. In particular, `skip_build` makes it just omit that `build` keyword so it just does `test`; it's not actually skipping the build step entirely, just the extra build.

`multi_scan` is interpreting `skip_build` in a different way. I'm not really sure precisely what it's doing, but from the code it looks like it's skipping batching of tests, and the output I get is a bit different.

Given that `multi_scan` already does a `build_for_testing` followed by a `test_without_building` by default, I'd have expected it to just ignore the `skip_build`.

**Comment by lyndsey-ferguson** [comment]
Okay, [here's](https://github.com/lyndsey-ferguson/fastlane-plugin-test_center/issues/41) where I introduced investigated this issue. I misunderstood what `:skip_build` does.  It seems that the proper fix is just to remove that option when it is calling `scan`.

**Comment by lyndsey-ferguson** [comment]
@lilyball, I believe I have addressed your concerns. Can you modify your `Pluginfile` per my instructions below, run `bundle install`, and then run your fastlane again (with the `--verbose` flag)?

Pluginfile:
```
gem 'fastlane-plugin-test_center', :git => "https://github.com/lyndsey-ferguson/fastlane-plugin-test_center.git", :branch => "issue-252-ignore-senseless-skip-build-option"
```

If there are still problems, please let me know and attach the console output as a text file to this issue (makes it easier for me to review).  If it works, please let me know.

**Comment by lyndsey-ferguson** [comment]
@lilyball I would appreciate it if you could test this: I'd like to confirm that this addresses your concerns. If you're too busy right now, I can merge this on Friday and we can open up another issue if there are remaining problems.

**Comment by lyndsey-ferguson** [comment]
Fixed in [v3.11.6](https://github.com/lyndsey-ferguson/fastlane-plugin-test_center/releases/tag/v3.11.6)
